.github/workflows/cargo.yml

```diff
@@ -118,7 +120,7 @@ jobs:
           - crate: swc
             os: ubuntu-latest
             check: |
-              cargo hack check --feature-powerset --no-dev-deps
+              cargo hack check --feature-powerset --no-dev-deps --exclude-features plugin_native_runtime
```

Comment:
Please modify https://github.com/swc-project/swc/blob/5800737752ec8a2befc7010290247b319e671ac3/scripts/github/tests.yml instead

Reply:
You can create matrix by running https://github.com/swc-project/swc/blob/5800737752ec8a2befc7010290247b319e671ac3/scripts/github/create-matrix.sh

Reply:
I tried to ran `create-matrix.sh` and nothing happens. What should be exactly updated to make sure script works?

Reply:
You should copy output to the matrix to this file

Reply:
Output doesn't seem to correct to me. Looks like script requires jq / yq, but after installing both I'm getting

```
...
parse error: Invalid numeric literal at line 1, column 6
parse error: Invalid numeric literal at line 1, column 6
parse error: Invalid numeric literal at line 1, column 6
- crate: testing_macros
  os: ubuntu-latest
parse error: Invalid numeric literal at line 1, column 6
parse error: Invalid numeric literal at line 1, column 6
parse error: Invalid numeric literal at line 1, column 6
- crate: wasm
  os: ubuntu-latest
parse error: Invalid numeric literal at line 1, column 6
parse error: Invalid numeric literal at line 1, column 6
parse error: Invalid numeric literal at line 1, column 6
```

Reply:
Are you using windows?
Interesting as it specifies `bash` requirement... I don't think `bash` parse script differently based on the OS

Reply:
No, it was linux and managed to made work - script requires to be executed cwd under `scripts`, does not allow `./scripts/create-matrix.sh` from swc's root. It wasn't clear as there wasn't way to see error outputs.